src/Input/Input.js

```diff
@@ -280,6 +302,7 @@ export default class Input extends Component {
           onFocus={this.handleFocus}
           onChange={this.handleChange}
           disabled={disabled}
+          defaultValue={defaultValue}
```

Comment:
Yea, that's a good point. So everything was working find with regular text fields based on an `<input />` but the defaultValue wasn't getting set for `<textarea>`'s until I explicitly passed defaultValue through as prop. 

This probably isn't the best solution... Any ideas on how to improve this? Or why it might not have been working via the `other` object for textareas?